client/src/common/types/Course.ts

```diff
@@ -3,6 +3,11 @@
  */
 interface Course {
     name: string;
+    courseID: string;
+    description: string;
+    credithours: number;
+    prereq:  string[];
+    satisfied_reqs: string[];
     title?: string;
```

Comment:
Isn't that the name?

Reply:
I didn't know what title was so I guess so.